app/controllers/users_controller.rb

```diff
@@ -12,7 +12,9 @@ def index
       mentoring:  'Helping as a Mentor',
       organizing: 'Helping as an Organizer'
     }
-    @users = User.ordered(params[:sort], params[:direction]).group('users.id').with_all_associations_joined
+    @users = User.ordered(params[:sort],
+                          params[:direction]).group('users.id').with_all_associations_joined.
```

Comment:
Putting a dot at the end of the line is super confusing as it's easy to miss, something along these lines is more readable:

```ruby
@users = User.ordered(params[:sort], params[:direction])
  .group('users.id')
  .with_all_assocations_joined
  .page(params[:page])
  .per(16)
```

It takes a few more lines but line count in itself isn't really an issue.